### PR TITLE
Create a Grafana Alert for expiring Microsoft Entra ID secrets

### PR DESCRIPTION
**Pull request by vanny96**

### CODE REVIEW

File: posts/2024-06-20-azure-entra-id-grafana-alerts.adoc [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+By the end of this section you'll need to know your App Registration's `<tenant-id>`, `<client-id>`
+and `<client-secret>`
+
+==== Create an App Registration
+
+. Login to Entra ID
+. Access the Microsoft Entra ID service
+. From the menu on the left, select the "App Registrations" blade
+. From the top, click "New Registration"
+. Fill the form (no rediraction URL is needed)
```
Comment (leo-bogastry) [inline posts/2024-06-20-azure-entra-id-grafana-alerts.adoc:52]:
```suggestion
. Fill the form (no redirection URL is needed)
```

**Comment by leo-bogastry** [comment]
Some of the images have a lot of white space. You should get a cropped version.

File: posts/2024-06-20-azure-entra-id-grafana-alerts.adoc [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+frequent rotation of those. This is very good from a security perspective, but caught my team and I by
+surprise on more than one occasion as suddenly our applications stop to work because of such secrets
+being expired.
+
+Unfortunately Azure doesn't seem to provide a built-in alert system for the expiring secrets, and the
+solutions we found online were not quite to our liking (https://www.thelazyadministrator.com/2023/12/16/automated-alerts-on-azure-entra-id-application-secret-expirations/[this excellent guide suggests a solution making
+use of powershell scripts] ) as we were hoping to fire the alerts from our Grafana instance since all
+the contact points were already configured.
+
+After a long investigation and some open source contributions, we finally arrived to the solution I'm
```
Comment (wjglerum) [inline posts/2024-06-20-azure-entra-id-grafana-alerts.adoc:24]: I would explicitly link to those contributions here!

File: posts/2024-06-20-azure-entra-id-grafana-alerts.adoc [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+. From the menu on the left, select the "App Registrations" blade
+. From the top, click "New Registration"
+. Fill the form (no rediraction URL is needed)
+. Click "Register"
+
+image::new_registrations.png[New Registration,600]
+
+==== Create an App Registration Secret
+
+. Open the App Registration object you just created (you should be automatically redirected there)
```
Comment (wjglerum) [inline posts/2024-06-20-azure-entra-id-grafana-alerts.adoc:59]: Can it also report for itself? Would be nice to add that explicitly here

**Review by leo-bogastry — DISMISSED** [review]

**Review by leo-bogastry — APPROVED** [review]

**Pull request merged**